### ISSUE TITLE
Usages not reported for ES6 destructuring in function signatures

### ISSUE THREAD

**Issue by use-strict** [Microsoft/TypeScript#7319]
**TypeScript Version:** 

1.8.2

**Code**

```ts
interface A {
    someProp: boolean;
}

function test({someProp}: A) {
    console.log(someProp);
}
```

**Expected behavior:**
When searching for usages of `someProp`, the destructuring in `test` function should count. I am using VSCode, but I understand this is a language server feature.

**Actual behavior:** 
The destructing statement in `test` function is not seen. Also, renaming with F2 doesn't affect this usage either.

**Comment by RyanCavanaugh** [comment]
Probably the same root cause as #6312 but leaving both open in case it's not

**Comment by sheetalkamat** [comment]
This is dupe of #6312 and gets also addressed as a test case in https://github.com/Microsoft/TypeScript/blob/renameAndFindRef/tests/cases/fourslash/renameDestructuringFunctionParameter.ts
